README.md

```diff
@@ -50,29 +50,454 @@ resolvers += "jitpack" at "https://jitpack.io"
+```
+
+You can also define the accumulation behavior of custom attributes:
+```scala
+div(
+  attr("everything").accum("-") := "im",
+  attr("everything").accum("-") := "possible",
+)
+// <div everything="im-possible"></div>
+```
+
+If you think there is something missing in Scala Dom Types, please open a PR or Issue. Usually it's just one line of code.
+
+Source Code: [DomTypes.scala](src/main/scala/outwatch/dom/DomTypes.scala)
+
+
+#### Data attributes
+Data attributes make use of [`scala.Dynamic`](https://www.scala-lang.org/api/current/scala/Dynamic.html), so you can write things like:
+
+```scala
+div(data.payload := "17")
+// <div data-payload="17"></div>
+```
+
+Source Code: [OutwatchAttributes.scala](src/main/scala/outwatch/dom/OutwatchAttributes.scala#L70), [Builder.scala](src/main/scala/outwatch/dom/helpers/Builder.scala#L39)
+
+
+#### SVG
+SVG tags and attributes are available via an extra import. Namespacing is automatically handled for you.
+
+```scala
+val graphic = {
+ import svg._
+ svg(
+   viewBox := "0 0 10 10",
+   g(
+     transform := "matrix(.096584 0 0 .096584 -.0071925 -18.66)",
+     path(d := "M10 10 C 20 20, 40 20, 50 10", fill := "mistyrose")
+   )
+ )
+}
+
+// <svg viewBox="0 0 10 10"><g transform="matrix(.096584 0 0 .096584 -.0071925 -18.66)"><path d="M10 10 C 20 20, 40 20, 50 10" fill="mistyrose"></path></g></svg>
+```
+
+**TODO**: Make an svg which actually shows something.
+
+Custom SVG attributes and tags are possible like this:
+
+```scala
+TODO: svgTag? svgAttr?
+```
+
+
+#### Option and Seq
+Outwatch tries hard to render everything you throw at it. Combine `Option` and `Seq` to fit your needs. Note, that outwatch does not accept `Set`, since the order is undefined.
+
+```scala
+div(
+  Some("thing"),
+  Some(color := "steelblue"),
+  fontSize :=? Some("70px"),
+  None,
+  Seq("Hey", "How are you?"),
+  List("a", "b", "c").map(span(_)),
+  Some(Seq("x")),
+)
+// <div style="color: steelblue; font-size: 70px;">
+//   thing
+//   Hey
+//   How are you?
+//   <span>a</span>
+//   <span>b</span>
+//   <span>c</span>
+//   x
+// </div>
+```
+
+Source Code: [AsVDomModifier.scala](src/main/scala/outwatch/AsVDomModifier.scala)
+
+
+
+#### Types
+The important types we were using in the examples above are `VNode` and `VDomModifier`:
+
+```scala
+val vnode:VNode = div()
+val modifiers:List[VDomModifier] = List("Hello", id := "main", color := "tomato", vnode)
+```
+
+Every `VNode` contains a sequence of `VDomModifier`. A `VNode` is a `VDomModifier` itself.
+
+There are implicits for converting primitives, `Option[VDomModifier]`, `Seq[VDomModifier]` to `VDomModifier`.
+
+
+#### Grouping Modifiers
+To make a set of modifiers reusable you can group them to become one `VDomModifier`.
+
+```scala
+val bigFont = VDomModifier(fontSize := "40px", fontWeight.bold)
+div("Argh!", bigFont)
+// <div style="font-size: 40px; font-weight: bold;">Argh!</div>
+```
+
+Of course you could also use a `Seq[VDomModifier]` instead.
```

Comment:
```suggestion
You can also use a `Seq[VDomModifier]` directly instead of using `apply` defined in the [VDomModifier](src/main/scala/outwatch/dom/package.scala) object.
```